packages/mobile/src/localization/languages/en.ts

```diff
@@ -11,4 +11,19 @@ export const en = {
     LAST_NAME: 'Last name',
     LAST_NAME_HINT: 'Enter your last name',
   },
+  currency: {
+    UAH: 'UAH',
+  },
+  button: {
+    BID: 'Bid',
+    BUY: 'Buy',
+  },
+  time: {
```

Comment:
I suppose we don't need it, cause we will use the library for if (F.e. dayjs) which supports localization. @AleksandrTermenzhy what do you think?